docs/bouwblokken/fundamentele rechten/non-discriminatie.md

```diff
@@ -1,44 +1,201 @@
 ---
-title: Non-discriminatie
-summary: hier staat een korte samenvatting over dit onderwerp
+title: Bias en non-discriminatie
 ---
+
 !!! info "Disclaimer"
 
     Het Algoritmekader is nog volop in ontwikkeling. Op deze plek willen we vooral aan de slag gaan op een open en transparante wijze. Het is dus niet definitief. Dat betekent dat er dingen opstaan die niet af zijn en soms zelfs fout. Mocht er iets niet kloppen, laat het ons weten via [GitHub](https://github.com/MinBZK/Algoritmekader).
 
-## Waarom?
-Schrijf hier een tekst over wat [title] is en waarom dit belangrijk is voor een verantwoord gebruik van algoritmes bij overheidsorganisaties. 
+## Wat en waarom?
+Algoritmes worden binnen de overheid veelvuldig ingezet om publieke taken uit te voeren. Dit biedt veel kansen, maar er zijn ook risico's aan verbonden.
+Zo kan bias[^1] in het algoritme leiden tot een ongelijke en oneerlijke behandeling van burgers of groepen, en kan er ook sprake zijn van discriminerende effecten. 
```

Comment:
Hoewel er voorbeelden zijn waarin algoritmes bias door mensen tegengaan, kan een algoritme ook leiden tot een ongelijke en oneerlijke behandeling van burgers of groepen, en kan er ook sprake zijn van discriminerende effecten. Wij richten ons hier op het laatste.

Reply:
zin 12 kan op basis hiervan ook aangepast worden: in dit bouwblok van het algoritmekader richten we ons op het laatste door aandacht te besteden aan de onderwerpen bias en non-discriminatie.